README.md

```diff
@@ -1,2 +1,385 @@
+
+## API Calls
+
+### GET
+
+#### Getting items on the menu
+
+Returns a JSON string of dishes (option for all dishes, and dishes by type). No authentication required.
+
+
+`GET /dishes/starters`  //returns starters
+
+
+`GET /dishes/mains`  //returns mains
+
+
+`GET /dishes/desserts`  //returns desserts
+
+
+`GET /dishes/refreshments`  //returns refreshments
+
+
+##### Sample Call
+	
+	fetch('http://localhost:3000/dishes')
+		.then (res => res.json())
+		.then ((data) => {
+			//do stuff with your data
+			})
+   	 })
+
+##### Success Response
+	
+	{
+		"success": true,
+		"message": "Requested dishes successfully retrieved.",
+		"status": 200,
+		"data": [{}, {}, {}]
+	}
+	
+##### Error Response
+
+	{
+		"success": false,
+		"message": "The resources requested do not exist at the desired location.",
+		"status": 404
+	}
+	
+#### Retrieving final order
+
+`GET /orders/:id`
+
+##### Sample Call
+
+	fetch('http://localhost:3000/orders/:id')
+		.then (res => res.json())
+		.then ((data) => {
+			//do stuff with your data
+			})
+   	 })
+
+##### Success Response
+
+	{
+		"success": true,
+		"message": "Final order successfully recieved",
+		"status": 200,
+		"data": [{}, {}, {}]
+	}
+
+##### Error Response
+
+	{
+		"success": false,
+		"message": "The resources requested do not exist at the desired location.",
+		"status": 404
+	}
+
+### POST: 
+
+#### Creating a new order
+
+`POST /orders`
+
+This endpoint allows you to create a new order.
+
+
+##### Data Params
+
+        {
+        	"name": "Ashley Coles",
+		"deliveryAddress": "BA2 6AH",
+		"email": "deliciousFood@food.com",
+        }
+
+
+##### Sample Call
+
+
+	fetch('http://localhost:3000/orders', {
+        	"method": POST,
+        	"body": JSON.stringify(/* your data goes here */),
+        	"headers": 
+		{
+            	"content-type": "application/JSON"
+        	}
+			.then (res => res.json())
+			.then ((data) => {
+				//do stuff with your data
+			})
+   	 })
+
+##### Success Response
+
+        {
+            "success": true,
+            "message": "Order created",
+            "status": 200,
+            "data": [
+                {
+		 	"_id": "60c73afb0b5f5c23d4a61688",
+			"name": "Ashley Coles",
+			"deliveryAddress": "BA2 6AH",
+			"email": "deliciousFood@food.com",
+			"isOrderSubmitted": false,
+			"timePlaced": "2000-01-01T00:00:00.000+00:00"
+			"orderItems": [{"menuItemId": "" , "quantity": "1"]
+                }
+            ]
+        }
+        
+
+##### Error Responses
+
+If the wrong datatypes are submitted
+
+         {
+            "success": false,
+            "message": "Must fulfil all required fields",
+            "status": 404
+        }
+		
+If the request fails to connect to the database
+
+         {
+            "success": false,
+            "message": "Database request failed",
+            "status": 404
+        }		
+
+
+### PUT
+
+#### Adding items to an order
+
+`PUT /orders/addToOrder`
```

Comment:
Could this just be a PUT to /orders? 

It would technically be more REST to do that, but I don't hate what you have